src/main/java/frc/robot/subsystems/Camera/Camera.java

```diff
@@ -13,10 +13,10 @@
 import org.photonvision.targeting.PhotonTrackedTarget;
 
 public class Camera extends SubsystemBase {
-  PhotonCamera camera;
+  static PhotonCamera camera;
 
   public Camera() {
-    this.camera = new PhotonCamera("slotheye");
+    Camera.camera = new PhotonCamera("slotheye");
```

Comment:
"Camera".camera references the constructor, while "this".camera references the entire class.  We need to be referencing the entire class.